dotnet/src/webdriver/DriverService.cs

```diff
@@ -386,4 +408,37 @@ private bool WaitForServiceInitialization()
 
         return isInitialized;
     }
+
+    private async Task ReadStreamAsync(StreamReader reader, string streamType)
+    {
+        try
+        {
+            string? line;
+            while ((line = await reader.ReadLineAsync()) != null)
+            {
+                if (streamType.Equals("stdout", StringComparison.OrdinalIgnoreCase))
+                {
+                    if (_logger.IsEnabled(LogEventLevel.Info))
+                    {
+                        _logger.Info(line);
+                    }
+                }
+                else
+                {
+                    if (_logger.IsEnabled(LogEventLevel.Error))
+                    {
+                        _logger.Error(line);
+                    }
+                }
+            }
+        }
+        catch (Exception ex)
+        {
+            if (_logger.IsEnabled(LogEventLevel.Error))
+            {
+                _logger.Error($"Error reading stream: {ex.Message}");
```

Comment:
```suggestion
                _logger.Error($"Error reading stream: {ex}");
```

Reveal the entire exception with stacktrace.